.github/workflows/run-code-tests.yaml

```
@@ -0,0 +1,56 @@
+      id-token: write
+    uses: ./.github/workflows/run-frontend-tests.yaml
+
+  run-e2e-tests:
+    name: E2E tests
+    permissions:
+      contents: read
+    uses: ./.github/workflows/run-e2e-tests.yaml
+
+  run-fuzz-graphql-tests:
+    name: GraphQL fuzz tests
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)